vitrina/smart_contracts/views.py

```diff
@@ -241,11 +242,14 @@ def get_success_url(self) -> str:
 
     @transaction.atomic
     def formset_valid(self, formset: BaseFormSet) -> HttpResponse:
+        current_user :User = self.request.user
```

Comment:
```suggestion
        current_user: User = self.request.user
```

You can also just run `poetry run ruff format`